app/views/multifactor_auths/otp_prompt.html.erb

```diff
@@ -1,11 +1,31 @@
 <% @title = t("multifactor_authentication") %>
 
-<%= form_tag @form_url, method: :post do %>
-  <div class="text_field">
-    <%= label_tag :otp, t("multifactor_auths.otp_code"), class: "form__label" %>
-    <%= text_field_tag :otp, "", class: "form__input", autofocus: true, autocomplete: :off %>
-  </div>
-  <div class="form_bottom">
-    <%= submit_tag t("authenticate"), data: {disable_with: t("form_disable_with")}, class: "form__submit" %>
-  </div>
-<% end %>
+<div class="mfa__container">
+  <% if @user.webauthn_credentials.any?%>
```

Comment:
Should this all be in the same view? Right now it's named `otp_prompt` but handles two cases.

Reply:
Yeah I guess this name isn't real appropriate anymore, I'll rename to `mfa_prompt` :)

Reply:
Done!